lib/main.dart

```diff
@@ -71,13 +67,12 @@ class _HomeState extends State<Home> {
       });
     }
   }
-
-  int _selectedIndex =0;
-
+  
+  int _selectedIndex = 0;
+  //we can also do it with var _selectedIndex = 0;
```

Comment:
cool, we can also do this even!

Reply:
Yep, thanks I'll update this in my local